### PR TITLE
Update for current release of desktop server

### PR DESCRIPTION
**Pull request by fperez**
As per @yuvipanda's [input](https://github.com/jupyterhub/jupyter-remote-desktop-proxy/pull/10#issuecomment-1397779175).

### CODE REVIEW

**Comment by fperez** [comment]
BTW, this is not urgent at all - yesterday's big update was all we needed for now, and this is more of a cleanup so we're up to date with the current tools.

**Comment by shaneknapp** [comment]
@fperez also curious if we should update `infra-requirements.txt` to match https://github.com/fperez/stat159-image/blob/main/infra-requirements.txt

**Comment by fperez** [comment]
No, we actually don't need that file at all - I'm cleaning up that other repo just now, hang tight :)

**Comment by shaneknapp** [comment]
> No, we actually don't need that file at all - I'm cleaning up that other repo just now, hang tight :)

word.  :)

**Comment by fperez** [comment]
All set @shaneknapp! Sorry for the slight delay, had a weird issue with getting the env to build locally (I double check that it works :) and it took a bit to sort out. All set!

**Comment by shaneknapp** [comment]
hey @fperez there are a few conflicts, so please take a look and let me know when they're done!

**Comment by fperez** [comment]
All conflicts fixed - they happened b/c folks were working on the files for this hub instead of a separate d100 one - in the future please LMK if you intend to work on these files so we can coordinate to avoid conflicts. I figured I'd be updating this folder alone :)

File: deployments/stat159/image/environment.yml [hunk 1]
```diff
@@ -76,22 +78,32 @@ dependencies:
...
+  - websockify==0.11.0
   - python-pdfkit==1.0.0 
 
+# Packages not available on conda-forge, installed through pip
   - pip:
-    - -r infra-requirements.txt
-    - jupyter-desktop-server==0.1.3
     - ipython-sql==0.4.1
     - jupyterlab_mystjs==0.1.1
-    - otter-grader==4.0.1
```
Comment (shaneknapp) [inline deployments/stat159/image/environment.yml:97]: ~~we'll still~~ do we need `otter-grader`...?

**Comment by shaneknapp** [comment]
> All conflicts fixed - they happened b/c folks were working on the files for this hub instead of a separate d100 one - in the future please LMK if you intend to work on these files so we can coordinate to avoid conflicts. I figured I'd be updating this folder alone :)

the conflicts happened as we were trying to get data100 unblocked w/otter-grader.

since we now have a separate data100 image, you'll be the only one (hopefully!) hacking on this folder moving forward.  :)

File: deployments/stat159/image/environment.yml (hunk 1 above)
Comment (shaneknapp) [reply deployments/stat159/image/environment.yml:97]: @fperez

**Comment by fperez** [comment]
Yup! This looks good to merge - I start teaching tomorrow, so if you have a chance to merge it, I'd appreciate it!

File: deployments/stat159/image/environment.yml (hunk 1 above)
Comment (shaneknapp) [reply deployments/stat159/image/environment.yml:97]: sorted via slack:  not necessary

**Review by shaneknapp — APPROVED** [review]

**Comment by fperez** [comment]
Great, thanks!!

**Pull request merged**